sdcm/sct_config.py

```diff
@@ -785,6 +785,11 @@ class SCTConfiguration(dict):
              env="SCT_K8S_SCYLLA_OPERATOR_UPGRADE_CHART_VERSION",
              type=str,
              help="Version of 'scylla-operator' Helm chart to use for upgrade."),
+        dict(name="k8s_scylla_operator_functional_test_dataset_size",
+             env="SCT_K8S_SCYLLA_OPERATOR_FUNCTIONAL_TEST_DATASET_SIZE", type=str,
+             help="Defines whether dataset size use for pre-fill cluster in functional test."
+                  "Expected values: small(219K), medium (111M), big (1G)."
+                  "Relevant if SCT_K8S_SCYLLA_OPERATOR_FUNCTIONAL_TEST_PREFILL_CLUSTER is True"),
```

Comment:
this comment isn't correct, i.e. there no `SCT_K8S_SCYLLA_OPERATOR_FUNCTIONAL_TEST_PREFILL_CLUSTER` 
and yes, we should be o.k. with only one configuration parameter for setting this up.

Reply:
forgot to remove it after parameter removing. fixed